bidb/api/urls.py

```diff
@@ -5,4 +5,6 @@
 urlpatterns = (
     url(r'^api/submit$', views.submit,
         name='submit'),
+    url(r'^api/buildinfo/since/(?P<date>[^/]+)$', views.buildinfo_since,
```

Comment:
Given that we know we want an integer here, I suggest:

```suggestion
    url(r'^api/buildinfo/since/(?P<date>\d+)$', views.buildinfo_since,
```